### ISSUE TITLE
question about training

### ISSUE THREAD

**Issue by crazy-stycxj** [ken2576/vision-nerf#21]
HI!  I repeat the training process,and find the loss like this ,Is this normal? It's like the whole workout was a shock at the end.
The whole process took 52 hours and the graphics card used was an RTX3090,24GB
Also can you give the code for the evaluation metrics for the test? thanks！

![image](https://github.com/ken2576/vision-nerf/assets/107168102/02d88f9c-db58-4099-9107-e31fd6e9a232)

**Comment by ken2576** [comment]
I think it looks ok. Usually with this kind of dataset, the loss would fluctuate heavily because the amount of unseen areas is different each sample.
The eval script should be here.
https://github.com/ken2576/vision-nerf/blob/main/eval.py

**Comment by crazy-stycxj** [comment]
> I think it looks ok. Usually with this kind of dataset, the loss would fluctuate heavily because the amount of unseen areas is different each sample. The eval script should be here. https://github.com/ken2576/vision-nerf/blob/main/eval.py

OK，thanks，I run the 【eval.py】。I find this code only output pics，not 【Assessment of indicators】，i mean how to comput 【PSNR,LILPS SSIM 】 like this

**Comment by crazy-stycxj** [comment]
> I think it looks ok. Usually with this kind of dataset, the loss would fluctuate heavily because the amount of unseen areas is different each sample. The eval script should be here. https://github.com/ken2576/vision-nerf/blob/main/eval.py

And I have one more question. You said in your paper that the decoder uses regular convolution, but I didn't find the corresponding part, can you point me to this part of the code, looking forward to your reply, thanks!

**Comment by ken2576** [comment]
Try the script from PixelNeRF to calculate the metrics. Maybe you need to change the paths to match the outputs.
https://github.com/sxyu/pixel-nerf/blob/master/eval/calc_metrics.py

I think the conv is defined here.
https://github.com/ken2576/vision-nerf/blob/main/network/vit.py#L468

**Comment by Lin666655** [comment]
Hello, I have deployed the environment strictly according to the file you provided, with CUDA version 11.1. I encountered the following error during runtime. What are the correct versions of PyTorch, torchvision, and cudatoolkit that I should use?
![2](https://github.com/user-attachments/assets/b482b0bd-215f-4e5d-9ed0-7d20f462da3f)
